doc/how_to_introduce_breaking_changes.md

```diff
@@ -149,10 +149,10 @@ register_argument_deprecate('bar foo', '--name', '--new-name')
 
 Declare breaking changes that affect the output of a command. This ensures users are aware of modifications to the command’s output format or content.
 
-* `command`: The name of the command group or command. If it is a command group, the warning would show in the execution of all commands in the group. 
-* `description`: The description of the breaking change. The description will display in warning messages.
-* `target_version`: The version when the deprecated item should be removed. The `target_version` is the next breaking change window by default.
-* `guide`: The guide that customers could take action to prepare for the future breaking change.
+* `command`: REQUIRED: The name of the command group or command. If it is a command group, the warning would show in the execution of all commands in the group. 
+* `description`: REQUIRED: The short-summary description of the breaking change. The description displays in warning messages.
+* `target_version`: REQUIRED: The version when the deprecated item should be removed. The `target_version` is the next breaking change window by default.
+* `guide`: REQUIRED: The migration guide that customers can follow to prepare for the future breaking change. Provide as much detail as possible to help our customers transition smoothly. This information is published in the [Azure CLI Breaking Changes]() article, but does NOT display in the warning message.
```

Comment:
`This information is published in the [Azure CLI Breaking Changes]() article, but does NOT display in the warning message`
Hi @dbradish-microsoft  I have not implemented this logic yet, but it is reasonable and I plan to include it in a future sprint. Should we merge this PR after the logic is implemented?

Reply:
@ ReaNAiveD , I feel this needs to be published now as it is our expectation. We need a place to send people that reflect our standards, regardless of if we have an azclibot to enforce them. make sense?

Reply:
Hi @dbradish-microsoft , 
I believe there will be no new breaking changes announced from now until Ignite, so it is okay to make this changes published aligned with the logic.

To clarify, this guidance is for the service team on how to announce and publish breaking changes. What we need your help with is publishing the customer-facing breaking change document, which I have shared with you in our Teams chat. And I think the latter one is much more urgent.

Reply:
`This information is published in the [Azure CLI Breaking Changes]() article, but does NOT display in the warning message`
After attempting to implement this and reconsidering, I believe it would bring significant changes to the current implementation of the breaking change report generation. Additionally, the warning message is more immediate and visible to users than the breaking change documents. Therefore, I think it is a good practice to include all necessary information in the warning message.

@dbradish-microsoft, could we remove all such declarations from the documentation? What do you think?

cc @Jacekey23

Reply:
@RealNAiveD, I would prefer if in-line help matched the information in the autogenerated BC article, so the removal of this sentence / process is good.